repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/serializer/AbstractDataSerializer.java

```diff
@@ -29,11 +41,16 @@ protected String getPrettyPrintState(ProjectInspector inspector) {
             return "test failure";
         }
 
-        if (jobStatus.isReproducedAsError()) {
-            return "test errors";
```

Comment:
Actually I took the decision to stop making difference between build with test failures and build with test errors: it's kind of confusing, the definition is not clear at all etc. So I now consider that if there was a test error/failure it was a test failure. Internally I kept the distinction in the metrics as it might be used for the repair tools.

Reply:
Omg, thank you.